scripts/utils.sh

```diff
@@ -10,11 +10,16 @@ function init_git_variables()
         export GIT_URL=https://github.com/snowflakedb/libsnowflakeclient.git
     fi
     if [[ -z "$GIT_BRANCH" ]]; then
-        BRANCH_NAME=$(git rev-parse --abbrev-ref HEAD)
-        export GIT_BRANCH=origin/$BRANCH_NAME
+        if BRANCH_NAME=$(git rev-parse --abbrev-ref HEAD); then
+            export GIT_BRANCH=origin/$BRANCH_NAME
+        else
+            echo "[WARN] No GIT_BRANCH is retrieved."
+        fi
```

Comment:
This is for the copy of source in the container. No GIT information is available.